server_installation/topics/operating-mode/crossdc/proc-configuring-remote-cache.adoc

```diff
@@ -0,0 +1,122 @@
+infinispan.client.hotrod.auth_server_name = infinispan
+infinispan.client.hotrod.sasl_mechanism = DIGEST-MD5
+infinispan.client.hotrod.trust_store_file_name = /path/to/truststore.jks
+infinispan.client.hotrod.trust_store_type = JKS
+infinispan.client.hotrod.trust_store_password = password
+----
++
+. Add `hotrod-client.properties` to the classpath of your {project_name} server.
+. Add a replicated cache named `work` to the Infinispan subsystem with the following configuration:
++
+----
+<replicated-cache name="work"> <1>
+    <remote-store cache="work" <2>
+                  remote-servers="remote-cache" <3>
+                  passivation="false"
+                  fetch-state="false"
+                  purge="false"
+                  preload="false"
+                  shared="true">
+        <property name="rawValues">true</property>
+        <property name="marshaller">org.keycloak.cluster.infinispan.KeycloakHotRodMarshallerFactory</property>
+    </remote-store>
+</replicated-cache>
+----
+<1> Names the cache in the {jdgserver_name} configuration.
+<2> Names the corresponding cache on the remote {jdgserver_name} cluster.
+<3> Specifies the `remote-cache` socket binding.
++
+The preceding cache configuration includes recommended settings for {jdgserver_name} caches.
+Hot Rod client configuration properties specify the {jdgserver_name} user credentials and SSL keystore and truststore details.
++
+Refer to the
+ifeval::[{project_community}==true]
+https://infinispan.org/docs/11.0.x/titles/xsite/xsite.html[{jdgserver_name} documentation]
+endif::[]
+ifeval::[{project_product}==true]
+https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.1/html-single/data_grid_guide_to_cross-site_replication/index#configure_clients-xsite[{jdgserver_name} documentation] 
+endif::[]
+for descriptions of each property.
+
+. Add distributed caches to the Infinispan subsystem for each of the following caches:
++
+* sessions
+* clientSessions
+* offlineSessions
+* offlineClientSessions
+* actionTokens
+* loginFailures
++
+For example, add a cache named `sessions` with the following configuration:
++
+----
+<distributed-cache name="sessions" <1>
+                   owners="1"> <2>
+    <remote-store cache="sessions" <3>
+                  remote-servers="remote-cache" <4>
+                  passivation="false"
+                  fetch-state="false"
+                  purge="false"
+                  preload="false"
+                  shared="true">
+        <property name="rawValues">true</property>
+        <property name="marshaller">org.keycloak.cluster.infinispan.KeycloakHotRodMarshallerFactory</property>
+    </remote-store>
+</distributed-cache>
+----
+<1> Names the cache in the {jdgserver_name} configuration.
+<2> Configures one replica of each cache entry across the {jdgserver_name} cluster.
+<3> Names the corresponding cache on the remote {jdgserver_name} cluster.
+<4> Specifies the `remote-cache` socket binding.
++
+. Start {project_name}.
```

Comment:
Missing exact steps for how to start KC instances. There should be something similar to the steps in the above Infinispan 9 section, or at least a reference to that section.

Reply:
Do you mean the steps to start a keycloak/SSO cluster?  Which Infinispan 9 section do you mean?